.gitignore

```diff
@@ -15,3 +15,6 @@ doc/_build
 *.pyo
 .idea/
 *.egg-info/
+# Files made by nose tests...
```

Comment:
Yeah, sorry this needed to be added. The proper solution would of course be to write temporary files to $TMPDIR (and delete them afterwards), but that’s something I’ll need to do.